java-extensions/hudi-reader/src/main/java/com/starrocks/hudi/reader/HudiScannerUtils.java

```diff
@@ -58,6 +60,11 @@ public static LocalDateTime getTimestamp(long value, TimeUnit timeUnit, boolean
         long nanoseconds = 0L;
 
         switch (timeUnit) {
+            case SECONDS:
+                seconds = value;
+                nanoseconds = 0;
+                break;
+
```

Comment:
where to use `SECONDS`?

Reply:
actually not used any. When I tested it, I've used that.